src/main/webapp/resources/js/components/galaxy/GalaxyDetailsForm.jsx

```diff
@@ -0,0 +1,75 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Checkbox, Form, Input } from "antd";
+import { getI18N } from "../../utilities/i18n-utilties";
+import { connect } from "react-redux";
+import { actions } from "./reducer";
+import { validateEmail } from "../../utilities/validation-utilities";
+
+/**
+ * Component to display a form containing all required and user modifiable fields.
+ * @param {string} email - email for galaxy
+ * @param  {boolean} makepairedcollection - whether to organize data into a library
+ * @param {function} updateEmail - handles updates to the email address
+ * @param {function} updateMakePairedCollection - handles update to makepairedcollection
+ * @returns {*}
+ */
+function ExportToGalaxyFormComponent({
```

Comment:
As part of this form, could you also display the URL to the Galaxy users are going to be redirected back into? That way, they know which Galaxy will get the files (say if there are multiple instances of Galaxy which connect to this IRIDA)?

Reply:
Updated in cc1c9031bd0213da64bb46ed12ddb594c57b0d9e.

![screen shot 2019-03-05 at 8 49 03 am](https://user-images.githubusercontent.com/11295750/53813615-a4097800-3f23-11e9-9f67-3039850c8760.png)

Reply:
Thanks, this looks awesome :smile: